### DISCUSSION TITLE
GithubEducation

### DISCUSSION THREAD

**Discussion by santosh2702** [community/community#20658] [GitHub Education] [answered]
How Can i Use GithubEducation Pack? if i am a Software Engineer.

**Comment by holiq** [comment, marked as answer]
if you are still a student then you can use github education, for how to get it, you can follow the method [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack) , and if it has been accepted you can take advantage of the student developer pack and use a pack that suits your work, and this list all offer [https://education.github.com/pack/offers](https://education.github.com/pack/offers)

**Reply by santosh2702** [reply]
ok Noted!

**Comment by Runner3131** [comment]
Like my friend above me said, if you're a student, then it'd be very worth your while. Even if you already knew full-stack programming, or what. 
Because you get a few 1000 dollars worth of interactive learning sites, software, visual studios, even jetbrains. Not to mention, you also get a couple different domain and hosting providers that offer you one or two domain names and free hosting while you're a student. One of them comes with the  .TECH tl. You also get a year free of Microsoft Azure Cloud. With 200 dollars credits that you don't have to use, because it comes with 25+ free services. The regular free trail is only good for 30 or 60 days. I don't remember, but the student one is that you keep your credits for the whole 12 months. Then you just have to re-show that you're still a student, which is very easy.

So yeah, if I were you, I'd go for the student benefit pack from Github, and the one from Microsoft. That way you can get some free certifications, that take time off your degree in some places.
But I got the feeling that you're not a student.
 Am I right?

But I couldn't be wrong.

**Reply by Runner3131** [reply]
We  hope you do. And if you ever need anything reach out.
